### PR TITLE
Add libusb to download_libs.sh and allow VS to include ofxKinect

### PR DESCRIPTION
**Pull request by ofTheo**

### CODE REVIEW

File: scripts/dev/download_libs.sh [hunk 1]
```diff
@@ -181,12 +181,15 @@ for PKG in $PKGS; do
...
+if [[ "$PLATFORM" == "osx" ]]; then
+    addonslibs=("opencv" "ippicv" "libusb" "assimp" "libxml2" "svgtiny" "poco" "openssl")
+    addons=("ofxOpenCv" "ofxOpenCv" "ofxKinect" "ofxAssimpModelLoader" "ofxSvg" "ofxSvg" "ofxPoco" "ofxPoco")
+elif [[ "$PLATFORM" == "ios" || "$PLATFORM" == "tvos" ]]; then
     addonslibs=("opencv" "ippicv" "assimp" "libxml2" "svgtiny" "poco" "openssl")
     addons=("ofxOpenCv" "ofxOpenCv" "ofxAssimpModelLoader" "ofxSvg" "ofxSvg" "ofxPoco" "ofxPoco")
 else
-    addonslibs=("opencv" "ippicv" "assimp" "libxml2" "svgtiny" "poco")
-    addons=("ofxOpenCv" "ofxOpenCv" "ofxAssimpModelLoader" "ofxSvg" "ofxSvg" "ofxPoco")
+    addonslibs=("opencv" "ippicv" "libusb" "assimp" "libxml2" "svgtiny" "poco")
```
Comment (arturoc) [inline scripts/dev/download_libs.sh]: i think this is not correct, only osx, vs and msys2 require libusb so there should be one more elif here for vs and msys2 that also includes libusb and then the current else for the remaining platforms

**Comment by arturoc** [comment]
also it seems there's still some errors with libusb in vs: 
```
c:\projects\openframeworks\addons\ofxkinect\libs\libfreenect\platform\windows\libusb10emu\libusb-1.0\libusbemu_threads.h(44): fatal error C1189: #error:  LIBUSBEMU PTHREAD WRAPPER NOT YET IMPLEMENTED! (compiling source file ..\..\..\addons\ofxKinect\libs\libfreenect\platform\windows\libusb10emu\libusb-1.0\failguard.cpp) [C:\projects\openframeworks\examples\templates\allAddonsExample\allAddonsExample.vcxproj]
C:\projects\openframeworks\addons\ofxKinect\libs\libfreenect\src\freenect_internal.h(203): warning C4200: nonstandard extension used: zero-sized array in struct/union (compiling source file ..\..\..\addons\ofxKinect\libs\libfreenect\platform\windows\libusb10emu\libusb-1.0\libusbemu.cpp) [C:\projects\openframeworks\examples\templates\allAddonsExample\allAddonsExample.vcxproj]
  C:\projects\openframeworks\addons\ofxKinect\libs\libfreenect\src\freenect_internal.h(203): note: This member will be ignored by a defaulted constructor or copy/move assignment operator (compiling source file ..\..\..\addons\ofxKinect\libs\libfreenect\platform\windows\libusb10emu\libusb-1.0\libusbemu.cpp)
```

**Comment by ofTheo** [comment]
Ah. Those errors will go away once the other PR is merged in, as libusb10emu is removed completely and not needed anymore. 

I'll fix the script as you mentioned. 
Thanks!!

File: scripts/dev/download_libs.sh (hunk 1 above)
Comment (ofTheo) [inline scripts/dev/download_libs.sh]: btw - I think its only vs and osx as msys2 uses pkg-config to add libusb-1.0.0 
pushing changes for that.

**Comment by ofTheo** [comment]
okay - I think this is good to go. 
needs #5706 to be merged to build properly.

**Comment by arturoc** [comment]
i've merged the 2 PRs in one to test that everything is working but osx is still failing. it seems like the library didn't have 64bit symbols? 

this is the merge PR: https://github.com/openframeworks/openFrameworks/issues/5711

and this is the ci logs: https://travis-ci.org/openframeworks/openFrameworks/jobs/258488095

**Comment by arturoc** [comment]
windows is working well now btw

**Comment by ofTheo** [comment]
@arturoc this should fix osx - forces universal 
https://github.com/openframeworks/apothecary/pull/77

strange as it was building universal for me before, but it could be that, that was set in an Xcode file not in the repo. the above should fix it.

**Pull request merged**